qa/test_framework/util.py

```diff
@@ -439,3 +440,46 @@ def get_coinbase_address(node, expected_utxos=None):
     addrs = [a for a in set(addrs) if addrs.count(a) == expected_utxos]
     assert(len(addrs) > 0)
     return addrs[0]
+
+"""
+Perform SC creation, mine mainchain blocks, create genesis info.
+Parameters:
+ - sidechain_id: id of the sidechain to be created
+ - mainchain_node: the mainchain node
+ - secrets: a JSON array of genesis secrets in the following form: [{"secret":"a string", "publicKey":"a string"}...]
+ - amounts: an array of amounts for each secret.
+ 
+Output: an array of two information:
+ - the genesis info used for start the sidechain node
+ - the height of the mainchain block at which the sidechain has been created (useful for future checks of mainchain block reference inclusion)
+
+"""
+def get_genesis_info(sidechain_id, mainchain_node, secrets, amounts):
+    number_of_blocks_to_enable_sc_logic = 219
```

Comment:
As i remember 200 block are already generated during mc network setup. Number of blocks can be increased there directly to avoid this step here.

Reply:
I should to check, I don't remember.

Reply:
We need to generate some blocks after mc network setup

Reply:
Do you mean, that after mc network was setup there is only a genesis block in the chain?